Source/Core/Core/GeckoCode.h

```diff
@@ -44,9 +44,22 @@ bool operator!=(const GeckoCode::Code& lhs, const GeckoCode::Code& rhs);
 constexpr u32 INSTALLER_BASE_ADDRESS = 0x80001800;
 constexpr u32 INSTALLER_END_ADDRESS = 0x80003000;
 constexpr u32 ENTRY_POINT = INSTALLER_BASE_ADDRESS + 0xA8;
-// If the GCT is max-length then this is the second word of the End code (0xF0000000 0x00000000)
-// If the table is shorter than the max-length then this address is unused / contains trash.
-constexpr u32 HLE_TRAMPOLINE_ADDRESS = INSTALLER_END_ADDRESS - 4;
+const u32 CODELISTSTART = 0x80002338;
+const u32 CODELISTEND = INSTALLER_END_ADDRESS;
```

Comment:
These should be constexpr.

Reply:
>  I feel should note that this seems to allow for code injection over netplay.

I don't think this allows for any more code injection over netplay than Gecko or AR Codes currently do, but I also am not an expert and could be missing something